ui/framework/src/ui-framework/content/DefaultViewOverlay.tsx

```diff
@@ -227,9 +227,8 @@ export class DefaultViewOverlay extends React.Component<Props, State> {
                 endDate={this.state.dataProvider.end}
                 initialDuration={this.state.dataProvider.initialDuration}
                 totalDuration={this.state.dataProvider.duration}
-                milestones={this.state.dataProvider.getMilestones()}
                 minimized={true}
-                alwaysMinimized={this.state.dataProvider.getMilestonesCount() > 0}
+                alwaysMinimized={true}

```

Comment:
do we even need to keep the alwaysMinimized prop?

Reply:
Removed.